Formula/hsd.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class Hsd < Formula
+  desc "Handshake Daemon & Full Node"
+  homepage "https://handshake.org"
+  url "https://github.com/handshake-org/hsd/archive/v2.1.2.tar.gz"
+  sha256 "4fc39ee5ca2b91186847a31b3244ae5d57235720ac40812cf752bb687c0833cf"
+
+  depends_on "python" => :build
+  depends_on "node@10"
+  depends_on "unbound"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    (bin/"hsd").write_env_script libexec/"bin/hsd", :PATH => "#{Formula["node@10"].opt_bin}:$PATH"
+  end
+
+  test do
+    (testpath/"script.js").write <<~EOS
+      const assert = require('assert');
+      const hsd = require('#{libexec}/lib/node_modules/hsd');
+      assert(hsd);
+    EOS
+    system "#{Formula["node@10"].opt_bin}/node", testpath/"script.js"
```

Comment:
As far as I can tell this only tests the existence of the install, not the validity. Please run some command or code and assert the output of that to see if it works.

Reply:
Thanks for the feedback, the test now better asserts that the software installed correctly. It creates its config directory in `script.js` and then the ruby asserts that the directory exists.